sqlutil/drivername.go

```diff
@@ -46,7 +46,7 @@ func DriverName(d driver.Driver) string {
 			return "postgresql"
 		}
 
-		if strings.HasPrefix(t.PkgPath(), "github.com/denisenkom/go-mssqldb") {
+		if strings.HasPrefix(t.PkgPath(), "github.com/microsoft/go-mssqldb") {
```

Comment:
Supporting the old prefix is tricky, because except if we change the driver name (which would be strange, as folks not using our instrumentation will not use the name we set here).
So I think releasing a minor version is what we need here.